components/event-gateway/internal/events/bus/process.go

```diff
@@ -0,0 +1,54 @@
+package bus
+
+import (
+	"fmt"
+
+	"github.com/kyma-project/kyma/components/event-gateway/internal/events/api"
+)
+
+type configurationData struct {
```

Comment:
I'm not certain what is the purpose of using pointer here, I think it adds some complexity. In main.go "&" operator is used to pass string's pointer to bus.Init. In bus.Init configurationData members are set but in AddSource * operator is used to dereference the pointer. I think bus.Init and configurationData could use string instead of *string (passing by value is more natural here).

Reply:
This is a code copy from the `gateway` component.